WordPress/src/main/java/org/wordpress/android/util/image/ImageManager.kt

```diff
@@ -96,6 +106,37 @@ class ImageManager @Inject constructor(val placeholderManager: ImagePlaceholderM
         return if (errorImageRes == null) this else this.error(errorImageRes)
     }
 
+    private fun GlideRequest<Drawable>.attachRequestListener(
+        requestListener: RequestListener?
+    ): GlideRequest<Drawable> {
+        return if (requestListener == null)
```

Comment:
Minor nitpick: I'd personally prefer to see the curly brackets here even though it's a single line, but I am not sure if we have a rule for it.

Reply:
I agree - I'm used to Java formatter which adds the brackets automatically.